packages/webpack/mixin.core.js

```diff
@@ -3,8 +3,8 @@ const { join } = require('path');
 
 const debug = require('debug')('untool:webpack:stats');
 const {
-  sync: { pipe, sequence },
-  async: { override },
+  sync: { pipe, sequence, override },
```

Comment:
I would prefer renaming `override` to `overrideSync`, similar to https://github.com/untool/untool/blob/master/packages/express/mixin.core.js#L2-L3.